### ISSUE TITLE
In IOS, the size of the marquee is not normal

### ISSUE THREAD

**Issue by 312307025** [aamirafridi/jQuery.Marquee#129]
In the case of IOS. The marquee, he would become much larger than normal. You can use an ordinary \<p\>xxxx\</p\> that doesn't roll to make a contrast. Obviously there is a problem.
My cell phone is iphone8plus, and the other colleagues' iPhone has this problem. I have no problem with Android.
The version of IOS is 11.2.5
My English is not very good, I'm sorry.
![_20180309173651](https://user-images.githubusercontent.com/17807336/37200689-7fe0c34a-23c0-11e8-8df5-498944fe92c0.jpg)

**Comment by nuke-ellington** [comment]
Thank you for sharing this. I ran the demo in the inspector emulating iPhone 8+ and was not able to reproduce it there. With no such device at my disposal right now could you maybe share additional info (e.g. code).

**Comment by 312307025** [comment]
@nuke-ellington 
I am sorry to reply you for such a long time.
I found that my iphone8plus updated the latest system.
The bug changed.
The font will grow larger at the beginning. After less than 1s, the font will return to its original form.
But I found it in the iphone6s phone that my colleague used.
The font will still grow larger.
Emmmm
I seem to have discovered the reason for this bug.
If you have a long speech, this will happen.
I enclosed my test code.
[test.1.txt](https://github.com/aamirafridi/jQuery.Marquee/files/2532429/test.1.txt)
